manifest.json

```
@@ -22,7 +22,7 @@
         "url": "https://github.com/mastereur"
     },
     "requirements": {
-        "yunohost": ">= 4.2.4"
+        "yunohost": ">= 4.3.5"
```

```suggestion
        "yunohost": ">= 4.3.4"
```

CI is not happy because not 100% up to date, but anyway 4.3.4 should be enough :stuck_out_tongue_winking_eye: